internal/keys/hasher.go

```diff
@@ -13,20 +13,20 @@ import (
 )
 
 type hasher interface {
-	WriteString(value string) error
+	WriteString(value string)
```

Comment:
the `error` being returned here was just forwarding along xxhash.WriteString `error`, which is always `nil`.